app/models/wiki_content/revision.rb

```diff
@@ -70,14 +70,19 @@ def plagiarism_report_link
 
   WIKITEXT_REF_TAGS = 'feature.wikitext.revision.ref_tags'
   WIKIDATA_REFERENCES = 'feature.len(<datasource.wikidatawiki.revision.references>)'
+  WIKI_SHORTENED_REF_TAGS = 'feature.enwiki.revision.shortened_footnote_templates'
 
   def ref_tags
-    features[WIKITEXT_REF_TAGS] || features[WIKIDATA_REFERENCES]
+    return nil if features.empty?
+    (features[WIKITEXT_REF_TAGS] || features[WIKIDATA_REFERENCES] || 0) +
```

Comment:
This is complicated enough that I think it makes sense to extract it to a method that can be used for both `features` and `features_previous`, instead of duplicating it.